apk_info/apk_info_test.go

```diff
@@ -0,0 +1,25 @@
+package apk_info
+
+import (
+	"path"
+	"runtime"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func Test_GetAPKPackageName(t *testing.T) {
+	// Given
+	_, filename, _, ok := runtime.Caller(0)
```

Comment:
you can refer to the test data with a path relative to this package since: when you run go test, for each package in scope, the test binary will be executed with its working directory set to the source directory of the package under test ([ref](https://dave.cheney.net/2016/05/10/test-fixtures-in-go)).